src/sbd-inquisitor.c

```diff
@@ -1307,37 +1303,52 @@ int main(int argc, char **argv, char **envp)
         } else if (strcmp(argv[optind], "ping") == 0) {
             exit_status = ping_via_slots(argv[optind + 1], servants_leader);
 
+        } else
+#endif
+        if (strcmp(argv[optind], "query-watchdog") == 0) {
+            exit_status = watchdog_info();
+        } else if (strcmp(argv[optind], "test-watchdog") == 0) {
+            exit_status = watchdog_test();
         } else if (strcmp(argv[optind], "watch") == 0) {
+            /* sleep $(sbd $SBD_DEVICE_ARGS dump | grep -m 1 msgwait | awk '{print $4}') 2>/dev/null */
+
+                const char *delay_source = delay ? "SBD_DELAY_START" : "";
+
+#if SUPPORT_SHARED_DISK
                 if(disk_count > 0) {
                     /* If no devices are specified, its not an error to be unable to find one */
                     open_any_device(servants_leader);
+
+                    if (delay_start && delay <= 0) {
+                        delay = get_first_msgwait(servants_leader);
+
+                        if (delay > 0) {
+                            delay_source = "msgwait";
+                            cl_log(LOG_WARNING, "Using 'msgwait' value for 'delay' starting instead of default '2 * watchdog-timeout'");
```

Comment:
Sorry if my unclear phrasing led to that ;-)
I guess we are in the branch here where everything runs as configured so no need for a warning.
I meant an else here that would warn about '2 * watchdog-timeout' being taken instead of unreadable 'msgwait'.

Reply:
No worries, actually this was my initial thought to. But I've understand the message a bit different.
Will fix that quick.